swift/ql/lib/codeql/swift/generated/File.qll

```diff
@@ -3,6 +3,6 @@ private import codeql.swift.generated.IpaTypes
 private import codeql.swift.generated.Db
 import codeql.swift.elements.Element
 
-class FileBase extends Cached::TFile, Element {
-  string getName() { result = asDbInstance().(Db::File).getName() }
+class FileBase extends Ipa::TFile, Element {
+  string getName() { result = Ipa::toDbInstance(this).(Db::File).getName() }
```

Comment:
Ruby calls these `toDbInstance` predicates `toGenerated`. Should we do the same?

Reply:
I'm not sure. We generate more than they do, including the synthesised layer, so while for them `Generated` unambiguously refers to stuff coming from the treesitter generation and that is in the dbscheme, for us the term would not be unambiguous. What do you think?